chat-widget/src/components/livechatwidget/LiveChatWidget.tsx

```diff
@@ -18,17 +21,42 @@ export const LiveChatWidget = (props: ILiveChatWidgetProps) => {
     const [state, dispatch]: [ILiveChatWidgetContext, Dispatch<ILiveChatWidgetAction>] = useReducer(reducer, getLiveChatWidgetContextInitialState(props));
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const [adapter, setAdapter]: [any, (adapter: any) => void] = useState(undefined);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const [facadeChatSDK, setFacadeChatSDK]: [any, (facade: any) => void] = useState(undefined);
     const chatSDK = getMockChatSDKIfApplicable(props.chatSDK, props?.mock?.type);
+
     overridePropsOnMockIfApplicable(props);
 
+    if (!props.chatConfig) {
+        throw new Error("chatConfig is required");
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const isAuthenticatedChat = ((props.chatConfig?.LiveChatConfigAuthSettings as any)?.msdyn_javascriptclientfunction)?true : false;
```

Comment:
Can be simplified as
```suggestion
    const isAuthenticatedChat = !!((props.chatConfig?.LiveChatConfigAuthSettings as any)?.msdyn_javascriptclientfunction);
```